packages/market-service/src/coingecko/coingecko.ts

```diff
@@ -18,20 +18,25 @@ type CoinGeckoAssetData = {
   }
 }
 
+const getId = (chain: string) => {
+  if (chain === ChainTypes.ETH) return 'ethereum'
```

Comment:
We should have an enum for these hard coded values as well.

Maybe we have a `ChainTypesShort` and `ChainTypesLong` and read the values from those.

Reply:
The ChainTypesLong would be specific to coingecko. So maybe not needed?

Reply:
If we do add ChainTypesLong it should be specific to the coingecko service. Maybe call it coingeckAssetId or something since its only used in coingecko urls

Reply:
yeah I agree with that. we can make an enum called `CoinGeckoIds` and define those

Reply:
I'd really prefer not to have a bunch of information about chains spread around in different enums, etc. There needs to be single source of truth.